system_test/common/aest_channels_SUITE.erl

```diff
@@ -292,8 +293,8 @@ reestablish_state_channel_perform_operations_close({INodeName, RNodeName},
     wait_for_value({txs_on_chain, [CloseTxHash]}, NodeNames, 5000, Config),
     %% wait for min depth to be reached so the channels die
     #{height := TopHeight} = aest_nodes:get_top(INodeName),
-    KeyBlocksToMine = 4 + 2, % min depth is 4
-    wait_for_value({height, TopHeight + KeyBlocksToMine}, NodeNames, 10000, Config),
+    KeyBlocksToMine = ?MIN_DEPTH + 2, % min depth is 4
+    wait_for_value({height, TopHeight + KeyBlocksToMine}, NodeNames, 20000, Config),
```

Comment:
So instead of waiting 10 seconds we wait 20 seconds? This is a very, very, very long time?!

Reply:
Fixed